R/utils-slope_selector.R

```diff
@@ -5,8 +5,8 @@
 #' @param data     Data to filter. Must be `PKNCAdata` list, containing the `conc` element with
 #'                 `PKNCAconc` list and appropriate data frame included under data.
 #' @param slopes   Data frame with slopes selection rules, must contain the following columns:
-#'                 `TYPE`, `PATIENT`, `PROFILE`, `RANGE`, `REASON`.
-#' @param profiles List with available profiles for each `PATIENT`.
+#'                 `TYPE`, `SUBJECT`, `PROFILE`, `RANGE`, `REASON`.
+#' @param profiles List with available profiles for each `SUBJECT`.
 #' @param slope_groups List with column names that define the groups.
```

Comment:
this is actually outdated anyway
```suggestion
#'                 `TYPE`,  `RANGE`, `REASON`, `slope_groups`
#' @param profiles List with available profiles for each `USUBJID`.
#' @param slope_groups List with column names that define the groups.
```